src/martialarts.cpp

```diff
@@ -471,6 +479,11 @@ std::string ma_requirements::get_description( bool buff ) const
                                type ) << std::endl;
     }
 
+    if( wall_adjacent ) {
+        dump << string_format( _( "* Can %s while <info>near</info> to a <info>wall</info>" ),
```

Comment:
```suggestion
        dump << _( "* Can %s while <info>near</info> to a <info>wall</info>" );
```
You're not formatting anything.

Reply:
This statement has the exact same format as the melee_allowed and unarmed_allowed statements a few lines above it. If they are all wrong then they all need to be changed but I'm pretty sure string_format is the "correct" way to do string substitution and sets up the string for proper language translation.

Reply:
> I'm pretty sure string_format is the "correct" way to do string substitution and sets up the string for proper language translation.

`string_format` does formatting via the embedded format specifiers like "%s". Without them, it just returns the input and is thereby completely useless. Same applies to the statements above (yes, the call to `string_format` is useless there).

Translating is done via the `_` function. Translation and string formatting are completely independent.

Reply:
Looks like all description text in martialarts.cpp uses `string_format`. Maybe it was done that way because of how it is concatenated  to `std::stringstream dump`? I have no idea but need to keep it the same or it might break something. If `string_format` is unneed, I think a new PR to remove all description references in martialarts.cpp should be made.

Reply:
Doesn't work without a `string_format`. It contains the `%s`. Newline required too.
![image](https://user-images.githubusercontent.com/1931904/64510196-17c07f00-d2d1-11e9-9d70-c80d6809bb5d.png)